### DISCUSSION TITLE
pintora plugin

### DISCUSSION THREAD

**Discussion by sombriks** [11ty/eleventy#3966] [Enhancement Queue]
### Is your feature request related to a problem? Please describe.

not a problem

### Describe the solution you'd like

would be nice a intora plugin: https://pintorajs.vercel.app/docs/getting-started/usage/

### Describe alternatives you've considered

is there a mermaid plugin? https://mermaid.js.org/syntax/classDiagram.html

**Comment by chriskirknielsen** [comment]
I haven't used it myself but there is a [Mermaid plugin by Kevin Gimbel](https://github.com/KevinGimbel/eleventy-plugin-mermaid) you can check out!

**Comment by zachleat** [comment]
Howdy! https://11tybundle.dev/ is a good place to keep up on community plugins!

It seems unlikely that we’ll add this to Eleventy core or as an official plugin but it would make a great community contribution! I’ll put it on the enhancement queue in case others disagree!

> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by sombriks** [comment]
@zachleat makes complete sense! i'll keep an eye on the enhancement queue.
